cypress/e2e/signIn.cy.js

```diff
@@ -17,18 +17,26 @@ describe('Sign In page', () => {
     });
   });
 
-  it('should provide an ability to log in with existing credentials', () => {
+  beforeEach(() => {
     signInPage.visit();
-    cy.register(user.email, user.username, user.password);
+  });
 
-    signInPage.typeEmail(user.email);
-    signInPage.typePassword(user.password);
-    signInPage.clickSignInBtn();
+  it.only('should provide an ability to log in with existing credentials',
+    () => {
+      cy.register(user.email, user.username, user.password);
```

Comment:
Ensure that the `cy.register()` command is correctly implemented and available in your Cypress commands. This command should handle the registration process using the provided user details. If it's not defined, the test might fail as it relies on this command to create a user before attempting to log in.